apps/vscode/plugins/jump-and-select.py

```diff
@@ -0,0 +1,18 @@
+from talon import Context, actions
+
+ctx = Context()
+ctx.matches = r"""
+    app: vscode
+    user.plugin_installed: arturodent.jump-and-select
+"""
+
+
+@ctx.action_class("user")
+class UserActions:
+    def jump_cursor_to_next_char(char: str):
```

Comment:
I wonder if these plugin implementations belong in the community repository. Perhaps we should consider moving knees into a separate plugin repository?

Reply:
Ya, plugins can get out of hand pretty fast, (see https://github.com/fidgetingbits/talon-vim/tree/main/vim/core/plugins) and the vast majority of people won't use most, so I can see the argument to move it somewhere else. I'm good with whatever.

Reply:
After pondering, I think this is one of the bridges we cross once it becomes unwieldy. We should at least have an example implementation in community. 🙂